docs/components/api-item-member.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import createFormatters from 'documentation/lib/output/util/formatters';
+import LinkerStack from 'documentation/lib/output/util/linker_stack';
+import docs from '../components/api.json'; // eslint-disable-line import/no-unresolved
+import ApiItem from '../components/api-item';
+import Icon from '@mapbox/mr-ui/icon';
+
+const linkerStack = new LinkerStack({})
+    .namespaceResolver(docs, (namespace) => {
+        const slugger = new GithubSlugger();
+        return `#${slugger.slug(namespace)}`;
+    });
+
+const formatters = createFormatters(linkerStack.link);
+
+class ApiItemMember extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {disclosed: false};
+        this.hashChange = this.hashChange.bind(this);
+    }
+
+    href = m => {
+        return `#${m.namespace.toLowerCase()}`;
+    }
+
+    render() {
+        const member = this.props;
+        return (
+            <div>
+                <div>
+                    <div className='pt60 border-b border--gray-light' style={{ marginTop: '-40px' }} id={member.namespace.toLowerCase()} />
+                    <div
+                        className='clearfix cursor-pointer toggle-sibling pt18'
+                        onClick={(e) => {
+                            this.setState({disclosed: !this.state.disclosed});
+                            window.location = this.href(member)                    
```

Comment:
I don't know if we want to set the location to the item when its heading in the main content is clicked because it causes the page to jump around in a surprising way (although this is the current behavior so feel free to ignore). I would expect the page to jump to the location only when a sidebar link was clicked.

Reply:
Yea. I'm not sure what the right answer is here. I don't want it to be to jarring to the reader, but we snap to headings when clicked in our other docs. 🤷‍♀️ But I suppose we could set a pattern where that's not true when the section is expandable. (Though we should probably make some way to easily grab the hash for the exact spot without having to find it on the sidebar.)